site-content/Dockerfile

```diff
@@ -99,7 +98,7 @@ ENV ANTORA_CONTENT_SOURCES_CASSANDRA_WEBSITE_BRANCHES="HEAD"
 ENV ANTORA_CONTENT_SOURCES_CASSANDRA_WEBSITE_TAGS=""
 ENV ANTORA_CONTENT_SOURCES_CASSANDRA_WEBSITE_START_PATH="site-content/source"
 
-ENV ANTORA_UI_BUNDLE_URL="https://github.com/ianjevans/antora-ui-datastax/releases/download/v0.1oss/ui-bundle.zip"
+ENV ANTORA_UI_BUNDLE_URL="https://github.com/apache/cassandra-website/raw/trunk/site-ui/build/ui-bundle.zip"
```

Comment:
don't we want the local path, i.e. `site-ui/build/ui-bundle.zip`?  (e.g. we've generated it locally)

Reply:
We could set the ui-bundle.zip to be the local path. However, such a change is has an unrealised complexity to it.

We would no longer need to change the Dockerfile, rather we would need change the `run.sh`. Specifically, the `run.sh` would need to change such that if `-z` is supplied as an argument that path is used, otherwise the local ui-bundle.zip file is mounted into the container. In either case, the `run.sh` would always set the value of the environment variable `ANTORA_UI_BUNDLE_URL`. Thus, defining a default value for it in the Dockerfile would be irrelevant.

Such a change would tightly couple the `run.sh` script to the Docker container. This is because the container would need to rely on external input for a file that is required by Antora to render the website. Currently on trunk, all critical files required by Antora have default values defined in the Dockerfile. This means, the container can run without any external input and render the website.

The current patch removes the need to specify the `-z` option so the user can rely on the container defaults to render the website.

Reply:
I'm thinking, if i `./run.sh` on a custom branch, i'd expect the `site-ui/build/ui-bundle.zip` to be used from that branch. It would not be intuitive that the file is there but not used and I have to specify the `-z` option.

Reply:
Ok. We I think we should leave the patch to the Dockerfile as is, and include the logic changes to the `run.sh` as well. This way the container can run without any external input, and `./run.sh` has the desired behaviour you have described.

Reply:
ah gotcha. if the logic change is so put into `/.run.sh` (i agree it should be), then that means the docker ENV will always be overridden when using the `/run.sh` approach. if that is the case, then i agree it makes sense to make the default ENV  ANTORA_UI_BUNDLE_URL point to the trunk url 👍

Reply:
Thanks @michaelsembwever. I have updated the `./run.sh` logic (which turned out to be easier than I thought) and pushed the changes to the branch.